nucleof072rb/Core/Src/main.c

```diff
@@ -143,7 +162,24 @@ void SystemClock_Config(void)
 }
 
 /* USER CODE BEGIN 4 */
+uint16_t handle_adc() {
+
+	// transmit data
+	uint8_t trans_data[3] = {0b00000001, 0b10000000, 0b00000000}; // first byte doesn't matter, 2nd byte need to target channel 1 single, 3rd byte doesn't matter
```

Comment:
I would recommend using hexadecimal representation to write those number. This way you can't mis-type a number and it will be easy to read.